src/NetAppFiles/NetAppFiles.Test/NetAppFiles.Test.csproj

```diff
@@ -7,12 +7,12 @@
   <Import Project="$(MSBuildThisFileDirectory)..\..\Az.Test.props" />
 
   <PropertyGroup>
-    <RootNamespace>$(LegacyAssemblyPrefix)$(PsModuleName)$(AzTestAssemblySuffix).ScenarioTests</RootNamespace>
+    <RootNamespace>$(LegacyAssemblyPrefix)$(PsModuleName)$(AzTestAssemblySuffix)</RootNamespace>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.NetApp" Version="0.9.1-preview" />
-    <PackageReference Include="Microsoft.Azure.Management.Network" Version="19.12.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.NetApp" Version="1.0.0" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="24.0.0-preview" />
```

Comment:
looks like the build is failing because of this version, is this the wrong version? @markcowl

Reply:
@tomasm9 the current version of `Microsoft.Azure.Management.Network` being used in our repository is `19.12.0-preview`